libraries/core-react/src/components/Input/Input.tsx

```diff
@@ -103,6 +98,7 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
 ) {
   const inputVariant = inputVariants[variant]
   const { density } = useEds()
+  const token = useToken({ density }, input)()
```

Comment:
last parameter here should be the `inputVariant` and not `input` as its whats used as "token" on line 108.

Reply:
`input` is the token that has `compact`, `inputVariant` only has tokens in regard to the four different variants.